packages/radix-ui-themes/src/components/reset.css

```diff
@@ -106,7 +108,7 @@
     border: none;
   }
   @supports (width: -webkit-fill-available) {
-    &:where(iframe, input, textarea) {
+    &:where(iframe, textarea) {
       width: -webkit-fill-available;
     }
```

Comment:
The original intention was to reset inputs so that they stretch to full width rather than use UA-defined width, however it doesn't work quite as expected when applied to non-text inputs. Decided to drop it for simplicity here.

Reply:
Makes sense!